src/OrbitGl/CallstackThreadBar.cpp

```diff
@@ -294,13 +294,12 @@ std::string CallstackThreadBar::GetSampleTooltip(const PrimitiveAssembler& primi
   std::string function_name = SafeGetFormattedFunctionName(*callstack, 0, -1);
   std::string result =
       absl::StrFormat("<b>%s</b><br/><i>Stack sample</i><br/><br/>", function_name.c_str());
-  if (callstack->type() == CallstackType::kComplete) {
-    result += "<b>Callstack:</b>" + FormatCallstackForTooltip(*callstack);
-  } else {
+  if (callstack->type() != CallstackType::kComplete) {
     result += absl::StrFormat(
-        "<b>Callstack not available:</b> the stack could not be unwound successfully.<br/>%s",
+        "<b>Unwinding error:</b> the stack could not be unwound successfully.<br/>%s",
         orbit_client_data::CallstackTypeToDescription(callstack->type()));
   }
+  result += "<br/><br/><b>Callstack:</b>" + FormatCallstackForTooltip(*callstack);
```

Comment:
I think we should try to make it clear that the callstack might be wrong if it's an error, like in the sampling tab. Is it possible to use a different color in the tooltips?

Reply:
With "clear" I mean "clearer", I think it would be useful to highlight the potentially wrong frames regardless of the "Unwind error" string.

Reply:
Done. http://screenshot/7M4cjeFzSNnDHbb